pipelined/bevy_render2/Cargo.toml

```diff
@@ -0,0 +1,59 @@
+[package]
```

Comment:
This appears to be a carry-over from before the switch to the new renderer on main.

Reply:
Fixed it